player/js/module.js

```diff
@@ -107,6 +111,7 @@
 
     bodymovinjs.play = play;
     bodymovinjs.pause = pause;
+    bodymovinjs.setLocationHref = setLocationHref;
```

Comment:
yes, you need to add it to the gulpFile.js in the variable moduleWrap.
I need to fix this because it's terrible for adding features, but it's something that doesn't change that often so it's never a priority. :/
If you want I can do this part myself.